### PR TITLE
Account Counterparts & paxgiro-credit

### PR DESCRIPTION
**Pull request by AntonCroner**

### CODE REVIEW

File: Transaction/index.ts [hunk 1]
```diff
@@ -13,7 +13,11 @@ import { Note as TransactionNote } from "./Note"
 import { Reference as TransactionReference } from "./Reference"
 import { Status as TransactionStatus } from "./Status"
 
-export interface Transaction extends Transaction.Creatable {
+export interface Transaction {
+	counterpart: Rail.Address
```
Comment (marcushanses) [inline Transaction/index.ts:17]: counterpart: Rail.Address & {code?: string}

File: Rail/Address/index.ts [hunk 2]
```diff
@@ -3,17 +3,19 @@ import { Card as AddressCard } from "./Card"
...
 	| AddressCard
 	| AddressCard.Counterpart
+	| AddressPaxgiroCredit
+	| AddressIban
+	| AddressInternal
+	| AddressPaxGiro
+	| AddressScan
 export namespace Address {
-	export const types = ["paxgiro", "internal", "iban", "scan", "card"] as const
+	export const types = ["paxgiro", "internal", "iban", "scan", "card", "funding"] as const
```
Comment (marcushanses) [inline Rail/Address/index.ts:18]: paxgiro-credit instead of funding.

File: Rail/Address/PaxgiroCredit.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export interface PaxgiroCredit {
+	source: "paxgiro"
+	type: "funding"
+	reference: string
+}
+export namespace PaxgiroCredit {}
```
Comment (marcushanses) [inline Rail/Address/PaxgiroCredit.ts:6]:
```suggestion
export interface PaxgiroCredit {
	type: "paxgiro-credit"
	reference: string
}
export namespace PaxgiroCredit {}
```

File: Transaction/index.ts (hunk 1 above)
Comment (AntonCroner) [reply Transaction/index.ts]: \+

File: Rail/Address/index.ts (hunk 2 above)
Comment (AntonCroner) [reply Rail/Address/index.ts]: \+

File: Rail/Address/PaxgiroCredit.ts (hunk 3 above)
Comment (AntonCroner) [reply Rail/Address/PaxgiroCredit.ts]: \+

**Review by marcushanses — APPROVED** [review]

**Pull request merged**